src/compiler/checker.ts

```diff
@@ -18864,28 +18862,34 @@ namespace ts {
                 }
             }
 
-            function getDeclarationSpaces(d: Declaration): SymbolFlags {
+            const enum DeclarationSpaces {
+                None = 0,
+                ExportValue = 1 << 0,
+                ExportType = 1 << 1,
+                ExportNamespace = 1 << 2,
+            }
+            function getDeclarationSpaces(d: Declaration): DeclarationSpaces {

```

Comment:
Never mind, I missed the call on line 18827 (search and context weren't easy to get on the bus). Now it makes sense.